src/format.ts

```diff
@@ -34,6 +34,7 @@ enum Design {
 	PhotoEssay,
 	PrintShop,
 	Obituary,
+	CorrectionsAndClarifications,
```

Comment:
Could this just be `Corrections`? Unless there's some existing history to these two appearing together (which is possible and if so, we should respect it) then adding clarifications seems redundant?

Reply:
Sure thing, I'll change it now.